KinWallet/UIKit Components/UIViewController/RootViewController.swift

```diff
@@ -327,7 +327,11 @@ extension RootViewController: NoticeViewControllerDelegate {
 
     func noticeViewControllerDidTapAdditionalMessage(_ viewController: NoticeViewController) {
         logClickedSupportOnErrorPage()
-        KinSupportViewController.presentSupport(from: viewController)
+        let helpCenterViewController = HelpCenterViewController()
+        helpCenterViewController.setPageToLoad(page: HelpCenterViewController.Page.support,
```

Comment:
Swift allows you to do the following:
`helpCenterViewController.setPageToLoad(page: .support, subCategory: .onboarding)`